src/components/Personalization/in-app-message-actions/utils.js

```diff
@@ -37,3 +39,38 @@ export const removeElementById = id => {
     element.remove();
   }
 };
+
+export const parseAnchor = anchor => {
+  const nothing = {};
+
+  if (!anchor || anchor.tagName.toLowerCase() !== "a") {
+    return nothing;
+  }
+
+  const { href } = anchor;
+  if (!startsWith(href, "adbinapp://")) {
```

Comment:
may be add one more check
`if (!href || !startsWith(href, "adbinapp://")) {
    return nothing;
}`